horizontal_rule_test.py

```diff
@@ -1,6 +1,54 @@
+import pytest
 from horizontal_rule import HorizontalRule
 
 
 def test_render():
     horiz_rule = HorizontalRule('')
     assert horiz_rule.render() == '<hr>'
+
+
+@pytest.mark.parametrize(
+    "content, start, end",
+    [
+        ('***', 0, 3),
+        ('____', 0, 4),
+        ('-----', 0, 5),
+        (' ------', 0, 7),
```

Comment:
1,7?

Reply:
I think 0,7 makes sense. no matter how may spaces, we turned the whole line into a `<hr>`, so theoretically, what we consumes is starting from the beginning of the line as the regex `^`

Also, the point of the indices was for other parser to continue parse this line. in this case, this line is always consumed as the whole line, so that the indices is pointless.